### PR TITLE
added content-cards-network-error-docs

### PR DESCRIPTION
**Pull request by radixdev**
# Pull Request/Issue Resolution

**Description of Change:**
Added docs for customizing the android error message for content cards refresh failure

Closes #**ISSUE_NUMBER_HERE**

### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Insert Feature Release Date Here__)
- [x] No

> If yes, please note the date of the feature release.

---
---

<details>
<summary>PR Checklist</summary>

- [ ] Check that all links work!
- [ ] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @KellieHawks as a reviewer when the your work is _done and ready to be reviewed for merge_.
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

</details>

<details>
<summary>Writing & Style Quick Tips</summary>

- Use the formatting shown on the [Styling Test Page](https://github.com/Appboy/braze-docs/blob/develop/_docs/_home/styling_test_page.md) to create unique templates and vehicles for your information.
- Use Action verbs!
- Capitalize proper nouns, unique Braze tools and features, custom Braze channels, and all Header titles.
- Redirect when necessary!
- Always speak to the Braze customer - refer to them as “you”, whether they are an Engineer/Developer, a Marketer, or an Admin.
- On Voice: You should aim to be a Friendly Professor. Imagine, if you will, the smartest, but also kindest professor you had. You know, the one who showed you your potential and tested you, but also went slowly in class so you could take great notes and learn some things? Don't be too peppy, but don't be formal. You’re not selling anything - you’re instructing and informing and enabling.

_Consult the [Docs Text Formatting Guide](https://github.com/Appboy/braze-docs/wiki/Special-Formatting) and the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/braze-docs/wiki/Writing-Style-Guide-&-Best-Practices) for more details._

</details>

<details>
<summary>ATTENTION: For PR Reviewers</summary>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
</details>

---
---

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/android/content_cards/customization.md [hunk 1]
```diff
@@ -53,6 +53,32 @@ If you would prefer, you can override these styles to create a look and feel tha
...
+  <item name="android:text">@string/com_appboy_feed_empty</item>
+  <item name="android:textColor">@color/com_appboy_title</item>
+  <item name="android:textSize">18.0sp</item>
+  <item name="android:gravity">center</item>
+  <item name="android:layout_height">match_parent</item>
+  <item name="android:layout_width">match_parent</item>
+</style>
+```
+
+To fully customize the network error behavior, you can extend the [AppboyContentCardsFragment][54] and set the `mShowNetworkUnavailableRunnable` variable to perform some other action.
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/content_cards/customization.md]: if you extent the fragment, how do you get the SDK to use the new class?

File: _docs/_developer_guide/platform_integration_guides/android/content_cards/customization.md [hunk 2]
```diff
@@ -53,6 +53,32 @@ If you would prefer, you can override these styles to create a look and feel tha
...
+
+This section covers customization of the [AppboyContentCardsFragment][49] whose source can be found [here][54].
+
+### Customizing The Network Connection Error Message
+
+If the [AppboyContentCardsFragment][49] determines that a Content Card refresh has failed, it will display a network connection error message.
+
+A special adapter, the [AppboyEmptyContentCardsAdapter][50] replaces the standard [AppboyCardAdapter][53] to display the error message.
+
+The style used to display this message can be found via [`Appboy.ContentCardsDisplay.Empty`][52] and is reproduced below.
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/content_cards/customization.md]: maybe note that the `com_appboy_feed_empty` string value can be changed to set the custom message text itself

File: _docs/_developer_guide/platform_integration_guides/android/content_cards/customization.md [hunk 3]
```diff
@@ -53,6 +53,32 @@ If you would prefer, you can override these styles to create a look and feel tha
 </style>
 ```
 
+## Customizing The Default Content Card Feed {#content-cards-fragment-customization}
+
+This section covers customization of the [AppboyContentCardsFragment][49] whose source can be found [here][54].
+
+### Customizing The Network Connection Error Message
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/content_cards/customization.md]: I think we should be clear in the header (and throughout the section) whether we're talking about customizing the message text or it's container (the adapter)

File: _docs/_developer_guide/platform_integration_guides/android/content_cards/customization.md (hunk 1 above)
Comment (radixdev) [reply _docs/_developer_guide/platform_integration_guides/android/content_cards/customization.md]: It's a fragment so clients already know how to use the extended class

File: _docs/_developer_guide/platform_integration_guides/android/content_cards/customization.md (hunk 2 above)
Comment (radixdev) [reply _docs/_developer_guide/platform_integration_guides/android/content_cards/customization.md]: done

File: _docs/_developer_guide/platform_integration_guides/android/content_cards/customization.md (hunk 3 above)
Comment (radixdev) [reply _docs/_developer_guide/platform_integration_guides/android/content_cards/customization.md]: Can you clarify what you mean? I don't understand

**Pull request merged**